api/v1beta1/ibmvpcmachine_types.go

```diff
@@ -36,10 +36,8 @@ type IBMVPCMachineSpec struct {
 	// Name of the instance.
 	Name string `json:"name,omitempty"`
 
-	// Image is the id of OS image which would be install on the instance.
-	// Example: r134-ed3f775f-ad7e-4e37-ae62-7199b4988b00
-	// TODO: allow user to specify a image name is much reasonable. Example: ibm-ubuntu-18-04-1-minimal-amd64-2
-	Image string `json:"image"`
+	// Image is the OS image which would be install on the instance.
+	Image *IBMVPCResourceReference `json:"image"`
```

Comment:
you shouldn't be changing type in the beta1, beta1 remains as is, we need to change only in the beta2 and add conversion to/from beta1

Reply:
my bad, fixing it.